workers/unity/Packages/com.improbable.gdk.core/Subscriptions/Systems/RequireLifecycleSystem.cs

```diff
@@ -9,8 +9,7 @@ namespace Improbable.Gdk.Subscriptions
     // or the order of events should be decided by system order rather than in this system
     // would enable users changing readers more easily maybe
     [DisableAutoCreation]
-    [UpdateInGroup(typeof(SpatialOSReceiveGroup.InternalSpatialOSReceiveGroup))]
-    [UpdateAfter(typeof(SpatialOSReceiveSystem))]
+    [UpdateInGroup(typeof(SpatialOSReceiveGroup.GameObjectInitializationGroup))]
```

Comment:
Should this also have `[UpdateAfter(typeof(GameObjectInitializationSystem))]`?

Reply:
We can't do that at the moment this assembly doesn't know about that one.
discussed offline - as these are both supposed to be different assemblies from core it might be best to do this via groups rather than by systems directly